### ISSUE TITLE
About the Forward Transfer

### ISSUE THREAD

**Issue by YizhaoGao-AI** [DonkeyShot21/cassle#11]
Hi,
Thanks for your excellent work!
I'm curious about how to calculate the "Forward Transfer" after training. For example, I have successfully re-produced the class-il results for Fine-tuning and CaSSLe (with BYOL) on Cifar-100 but don't know how to directly check the FT results. Does it need a seperate run to obtain the "linear evaluation accuracy of a random network" as the paper stated? 
BTW, just to be sure, is it right to directly check the "val_acc1" results of wandb board as the final linear evaluation accuracy?

**Comment by DonkeyShot21** [comment]
Hi. Thanks for the interest in our work. 

Yes, if I remember correctly I ran linear evaluation on a random network. 

Yes, it is fine to use the val_acc1 for cifar, while for in100 and DomainNet you should run linear evaluation (for all intermediate tasks as well).

**Comment by YizhaoGao-AI** [comment]
Got it, thanks again!
